html_css/assets/css/style.css

```diff
@@ -0,0 +1,1091 @@
+.bg-green-bottom {
+  position: absolute;
+  top: 0;
+  right: 0px;
+}
+
+.bg-green-top {
+  z-index: 2;
+  max-width: 740px;
+}
+
+.bg-green-bottom {
+  max-width: 778px;
+}
+
+header {
+  z-index: 4;
+  padding-top: 44px;
+  margin: 0 auto;
+  max-width: 850px;
+  height: 40px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.svg-union-logo {
+  z-index: 4;
+  cursor: pointer;
+  fill: #1B63F0;
+}
+
+.svg-union-logo:hover {
+  fill: black;
+}
+
+header nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+header nav ul {
+  display: flex;
+  z-index: 4;
+  justify-content: space-between;
+}
+
+header nav button {
+  width: 96px;
+  z-index: 4;
+}
+
+header nav ul li a {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 1.33;
+  color: #fff;
+}
+
+header nav ul li {
+  margin-right: 20px;
+}
+
+header nav ul li:last-child {
+  margin-right: 40px;
+}
+
+header nav ul li a:hover {
+  color: #1B63F0;
+}
+
+
+.weapon {
+  max-width: 850px;
+  margin: 0 auto;
+  display: flex;
+}
+
+.weapon article {
+  display: flex;
+  flex-direction: column;
+}
+
+.weapon h1 {
+  z-index: 4;
+  margin-top: 155px;
+  margin-bottom: 20px;
+}
+
+.weapon p {
+  z-index: 4;
+  margin-bottom: 40px;
+}
+
+.weapon img {
+  z-index: 3;
+  width: 100%;
+  max-width: 403px;
+  max-height: 403px;
+  margin-top: 76px;
+  margin-right: -4px;
+}
+
+.weapon__buttons {
+  display: flex;
+  align-items: baseline;
+}
+
+.weapon__buttons a {
+  margin-left: 24px;
+  padding: 12px 24px;
+  width: 119px;
+  height: 40px;
+  background: #97F5CD;
+  border-radius: 22px;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 1.33;
+  color: #000000;
+}
+
+.weapon__buttons a:hover {
+  background: #C0FFE4;
+}
+
+.weapon__buttons a:active {
+  background: #83EBBF;
+}
+
+
+.weapon__links {
+  display: flex;
+  align-items: center;
+  padding-top: 23px;
+}
+
+.weapon__links img {
+  margin-right: 0px;
+}
+
+
+.choose {
+  max-width: 850px;
+  margin: 0 auto;
+  padding-top: 64px;
+  display: flex;
+  justify-content: space-between
+}
+
+.choose__left svg {
+  width: 56px;
+  height: 56px;
+}
+
+.choose__left h2 {
+  padding-bottom: 33px;
+}
+
+.choose__left h2::after,
+.tour article h2::after,
+.prefer h2::after,
+.customers__feedback h2::after,
+.blog h2::after,
+.choose-two article h2::after {
+  content: "";
+  display: inline-block;
+  border-bottom: 2px solid #1B63F0;
+  width: 60px;
+  margin-left: 25px;
+}
+
+
+.choose__left--line1 {
+  display: flex;
+  justify-content: space-between;
+}
+
+.choose__left--line1 article {
+  display: flex;
+  flex-direction: column;
+  padding-top: 5px;
+}
+
+.choose__left--line1 h3,
+.choose__left--line2 h3 {
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 1.5;
+  color: #000000;
+  margin-top: 17px;
+  margin-bottom: 0;
+}
+
+.choose__left--line1 p,
+.choose__left--line2 p {
+  margin-top: 6px;
+}
+
+.choose__left--line1Article2 {
```

Comment:
Class names has no Uppercase letters, spaces, translit and tag names, only english letters and words. if Class name has more than one word, split it with hyphen

Reply:
corrected